### DISCUSSION TITLE
OCA Runway success logic

### DISCUSSION THREAD

**Discussion by DevastatorCM** [Khopa/dcs_liberation#2179] [Q&A] [answered]
Hi Guys,
what is the logic of successfull Oca strike? Is there any threshold of total bomb sizes (i.e. 2000lbs should hit which can be 4 MK82 or 1 mk84) or does it count the bomb amounts which hit and their types?

On top of that, how does it work if the airfield has multiple runways?

Thanks in advance

**Comment by Mustang-25** [comment, marked as answer]
There's a damage threshold you need to cross for DCS to generate the Runway Dead event. 1x GBU-31v3 will do it and iirc it's something like 5 mk-84s. The way Liberation is structured right now, even with multiple runways, destroying one will put that airbase out of commission.

**Reply by Drofseh** [reply]
Do you know how many BLG 66 or BAP-100 from the M-2000C are required to generate the Runway Dead event?

**Reply by Drofseh** [reply]
> BAP-100

It's 36, so two strikes of 18.

**Comment by DevastatorCM** [comment]
Thanks for the info. U guess you mean 5 mk 82 and not 84.
Is this functionality can be improved (type of the ammunition, its fuse settings -i.e. penetrator vs not penetrator-, its warhead size, runway length -1 bomb in middle might still not stop take offs for small airplanes at long runways- etc.) or due to engine limitations cannot be?

**Reply by Mustang-25** [reply]
No, I meant what I said. Mk-84s. 82s take about 14 of them.

**Reply by Starfire13** [reply]
Runway destruction is handled by DCS, not Liberation. Liberation simply records the event.

**Comment by Simba75013** [comment]
6000lbs of general purpose explosive (12x Mk82) or 1x 2000lbs penetrator. My weapon of choice is the GBU 31v4.
I usually drop 2 of them for safety but 1 does the job.

Quite some daring attack but a loft release works great.
